webcheck.go

```diff
@@ -144,29 +142,33 @@ func (fm *Frontman) runWebCheck(data WebCheckData) (m map[string]interface{}, er
 	}
 	defer resp.Body.Close()
 
+	// wrap ReadCloser with the ReadCloserCounter
+	bodyReaderWithCounter := datacounters.NewReadCloserCounter(resp.Body)
```

Comment:
You can write this in one line. No need for `bodyReaderWithCounter `

Reply:
I need it later to get a counter from the `ReadCloseCounter` struct, otherwise, I need to make a type assertion for the interface

Reply:
✅